src/Chullo.php

```diff
@@ -226,21 +232,23 @@ public function saveResource(
     public function saveGraph(
         $uri,
         \EasyRdf_Graph $graph,
+        $checksum_algorithm = "",
         $transaction = ""
     ) {
         // Serialze the rdf.
         $turtle = $graph->serialise('turtle');
 
         // Checksum it.
-        $checksum = sha1($turtle);
+        $checksum_value = sha1($turtle);
 
         // Save it.
         return $this->saveResource(
             $uri,
             $turtle,
             ['Content-Type' => 'text/turtle'],
             $transaction,
-            $checksum
+            $checksum_value,
+            $checksum_algorithm
```

Comment:
You also don't need this line, it will default to "sha1" which is what you are using.